info/metadata_tags.md

```diff
@@ -21,4 +21,26 @@ Standard tags are defined in MetadataKeyConstants
 | TRADEMARK     | Trademark    | A trademark declaration if available | String e.g. "Splunge is a registered trademark of ByteStyle" |
 | ARTIFACTID    | ArtifactID   | An identifier for the artifact | String |
 | LICENSE       | License      | The lisence for the artifact | String e.g. "This work is openly licensed via CC BY 4.0" |
+| DEPENDENCIES  | Dependencies | A list of the dependencies | String, formatted as JSON. See below. |
```

Comment:
Metadata values can be represented either as a `String` or `(String, String)` (tuple of string) where the first string is the mime type and the second is the value.

It strikes me that explicitly adding the mime type here will make explicit what the implicit/documented type is

Reply:
Understood.